bool sim
int16 planeID
float64 latitude
float64 longitude
float64 altitude
---
string error
